resources/js/components/SignIn.js

```diff
@@ -1,292 +1,250 @@
-                    <DialogTitle>
-                        Signing In...
-                        <CircularProgress
-                            color="primary"
-                            sx={{
-                                float: "right",
-                                marginRight: "90px",
-                                marginTop: "-5px",
-                            }}
-                        ></CircularProgress>
-                    </DialogTitle>
-                </Dialog>
-            ) : null}
-            <Grid
-                item
-                xs={false}
-                sm={4}
-                md={7}
-                sx={{
-                    backgroundImage: "url(images/bg-6.jpeg)",
-                    backgroundRepeat: "no-repeat",
-                    backgroundColor: (t) =>
-                        t.palette.mode === "light"
-                            ? t.palette.grey[50]
-                            : t.palette.grey[900],
-                    backgroundSize: "cover",
-                    backgroundPosition: "center",
-                }}
-            ></Grid>
-            <Grid
-                item
-                xs={12}
-                sm={8}
-                md={5}
-                component={Paper}
-                elevation={6}
-                square
-            >
-                <Box
-                    sx={{
-                        my: 8,
-                        mx: 4,
-                        display: "flex",
-                        flexDirection: "column",
-                        alignItems: "center",
-                    }}
-                >
-                    <Avatar
-                        sx={{
-                            width: "200px",
-                            height: "200px",
-                            marginBottom: "-30px",
-                            marginTop: "-50px",
-                        }}
-                        src="images/e-learning.png"
-                    ></Avatar>
-                    <Typography
-                        component="h1"
-                        variant="h5"
-                        sx={{ marginTop: "-40px" }}
-                    >
-                        Sign in
-                    </Typography>
-                    <Typography
-                        component="h6"
-                        variant="h6"
-                        style={{ color: "red" }}
-                    >
-                        {props.requestErrorMessage}
-                        &nbsp;
-                    </Typography>
-                    <Box
-                        component="form"
-                        noValidate
-                        onSubmit={handleSubmit}
-                        sx={{ marginTop: "10px" }}
+  // Toggle Show Password
+  const handleShowPassword = () => {
+    props.showPassword(props.isShownPass);
+  };
+
+  return (
+    <Grid container component="main" sx={{ height: '100vh' }}>
+      {props.requestError === false ? (
```

Comment:
```suggestion
      {props.requestError ? (
```

Reply:
removing the `=== false` will make the Sign In dialog not show.

Reply:
@michaelsilverio oki disregard the comment na lang